functions/exercise2/script.js

```diff
@@ -1,10 +1,20 @@
 "use strict";
-
 function isPalindrome(phrase) {
     var normalizedPhrase = phrase.toLowerCase().trim();
     if (normalizedPhrase === "") {
         throw new Error("Parameter must contain at least one letter - not use white space");
     }
     var reversedNormalizedPhrase = normalizedPhrase.split("").reverse().join("");
     return reversedNormalizedPhrase === normalizedPhrase;
+}
+
+function showResult() {
+    var phrase = document.getElementById("phrase").value;
+    try {
+        var result = isPalindrome(phrase);
+        document.getElementById("result").value = `Phrase is palindrome: Is ${result}`;
```

Comment:
Why can't you simply display either information that specific phrase is a palindrome or is not a palindrome?
Does phrase: "Phrase is palindrome: Is true" is ok to be displayed to end user?
When I put word dog here I expect having info:
"'dog' is not a palindrome"
When I put workd kayak I expect having info:
"kayak is a palindrome"